dags/fivetran_acoustic.py

```diff
@@ -10,6 +10,9 @@
 
 
 def _generate_acoustic_report(conn_id, report_type, config, *args, **kwargs):
+    if config["request_params"]["date_start"] == config["request_params"]["date_end"]:
+        raise ValueError("It appears start and end date are exactly the same.")
```

Comment:
You might consider a more explanatory sentence of the expected behaviour, for example: "The start and end date cannot be the same, please check."